### ISSUE TITLE
VS Code: inline quokka text & background color workbench settings?

### ISSUE THREAD

**Issue by metasean** [wallabyjs/quokka#83]
### Issue description or question

tldr - What are the VS Code settings to change the text and background color of the inline quokka reports?

Until Support for Atom #20 is available, I'm fighting VS Code.  One of the things that's driving me bonkers is that with my color theme, the inline quokka feedback is too visually similar to inline comments.  Otherwise, the theme is usable and I like the comments the way they are, so I just want to change the text and background color of the inline quokka reports, however I really don't want to spend time becoming an expert in [VS Code Theme Settings](https://code.visualstudio.com/docs/getstarted/theme-color-reference) to do so.  

(And please hurry with the Atom plugin, I'm really digging quokka, but VS Code is going to lead to the violent destruction of my computer and me relocating to a funny farm.)

### Code editor version

Visual Studio Code v1.15.

### OS name and version

OSX

**Comment by ArtemGovorov** [comment]
> What are the VS Code settings to change the text and background color of the inline quokka reports?

It's not possible at the moment, we'll look into making it configurable.

> And please hurry with the Atom plugin, I'm really digging quokka

It's very close, stay tuned.

**Comment by metasean** [comment]
@ArtemGovorov - Thanks for such a fast response!

I'm sad to hear that we can't make the VS Code colors more usable, but really giddy for the Atom version of quokka!

**Comment by ArtemGovorov** [comment]
> I'm sad to hear that we can't make the VS Code colors more usable

Oh, we can, it's just that the extension doesn't yet support it. So I'm re-opening this as a valid feature request.

**Comment by bluelovers** [comment]
hope can for JetBrains IDEs too.

i see quokka in color scheme, but no options for inline color

**Comment by ArtemGovorov** [comment]
@bluelovers Yes, [you can do it](https://github.com/wallabyjs/quokka/issues/188#issuecomment-384113415).

**Comment by eadorin** [comment]
Just want to give this a bump. @ArtemGovorov mentioned that it uses the "System Console" setting...but that doesn't exactly exist on Linux. I really need to be able to adjust this (To match my comment color, maybe?) as right now it is too close to my regular code and is super distracting. (I'm colorblind so it's extra important for someone like me). Thanks!

**Comment by smcenlly** [comment]
The JetBrains color that you are looking for is called `System output` (as per @ArtemGovorov's original reply). I can confirm that this is available on Linux builds of IntelliJ Editors. Please see screenshot below:

![](https://user-images.githubusercontent.com/2074089/145755666-e5b7aba2-3e1d-4125-aa27-51051e1452d2.png)

Please note that you will need to restart your editor for the color change to take effect.

**Comment by eadorin** [comment]
> The JetBrains color that you are looking for is called `System output` (as per @ArtemGovorov's original reply). I can confirm that this is available on Linux builds of IntelliJ Editors. Please see screenshot below:
> 
> ![](https://user-images.githubusercontent.com/2074089/145755666-e5b7aba2-3e1d-4125-aa27-51051e1452d2.png)
> 
> Please note that you will need to restart your editor for the color change to take effect.

I'm sorry I didn't specify, I'm using Visual Studio Code, not IntelliJ.

**Comment by ArtemGovorov** [comment]
@eadorin You may change Quokka VS Code color settings as [described here](https://github.com/wallabyjs/quokka/issues/83#issuecomment-325057573).

**Comment by metasean** [comment]
<img width="554" alt="Screen Shot 2021-12-13 at 4 59 46 PM" src="https://user-images.githubusercontent.com/7552754/145908407-8c07be4b-21d3-4803-8685-69412bab91b5.png">


Apologies.  I didn't actually post anything to GitHub 19 hours ago, so I'm guessing someone hacked my account. 😡  I've changed my password, so hopefully it doesn't happen again.

**Comment by ArtemGovorov** [comment]
@metasean Oh, there wasn't any issue with your comment and I think your account is safe. I have just been clearing the issue and deleted a few obsolete comments (yours and mine) in preparation to close the issue (because we'll be soo releasing the settings to change VS Code colors for Quokka).

**Comment by smcenlly** [comment]
@metasean - You can override the color of the inline values with the VSCode settings below

**Quokka package.json settings**
```javascript
"quokka.lightTheme.log.decorationAttachmentRenderOptions": {
  "description": "Override Quokka Light Theme Log",
  "type": "object",
  "default": {
    "border": null,
    "borderColor": null,
    "fontStyle": null,
    "fontWeight": null,
    "textDecoration": null,
    "color": "#0000ff",
    "backgroundColor": null,
    "margin": "1.2em",
    "width": null,
    "height": null
  }
},
"quokka.lightTheme.system.decorationAttachmentRenderOptions": {
  "description": "Override Quokka Light Theme System Log",
  "type": "object",
  "default": {
    "border": null,
    "borderColor": null,
    "fontStyle": null,
    "fontWeight": null,
    "textDecoration": null,
    "color": "rgb(153, 153, 153)",
    "backgroundColor": null,
    "margin": "1.2em",
    "width": null,
    "height": null
  }
},
"quokka.lightTheme.error.decorationAttachmentRenderOptions": {
  "description": "Override Quokka Light Theme Error",
  "type": "object",
  "default": {
    "border": null,
    "borderColor": null,
    "fontStyle": null,
    "fontWeight": null,
    "textDecoration": null,
    "color": "#c80000",
    "backgroundColor": null,
    "margin": "1.2em",
    "width": null,
    "height": null
  }
},
"quokka.darkTheme.log.decorationAttachmentRenderOptions": {
  "description": "Override Quokka Dark Theme Log",
  "type": "object",
  "default": {
    "border": null,
    "borderColor": null,
    "fontStyle": null,
    "fontWeight": null,
    "textDecoration": null,
    "color": "rgba(86, 156, 214, 1)",
    "backgroundColor": null,
    "margin": "1.2em",
    "width": null,
    "height": null
  }
},
"quokka.darkTheme.system.decorationAttachmentRenderOptions": {
  "description": "Override Quokka Dark Theme System Log",
  "type": "object",
  "default": {
    "border": null,
    "borderColor": null,
    "fontStyle": null,
    "fontWeight": null,
    "textDecoration": null,
    "color": "rgb(153, 153, 153)",
    "backgroundColor": null,
    "margin": "1.2em",
    "width": null,
    "height": null
  }
},
"quokka.darkTheme.error.decorationAttachmentRenderOptions": {
  "description": "Override Quokka Dark Theme Error",
  "type": "object",
  "default": {
    "border": null,
    "borderColor": null,
    "fontStyle": null,
    "fontWeight": null,
    "textDecoration": null,
    "color": "#fe536a",
    "backgroundColor": null,
    "margin": "1.2em",
    "width": null,
    "height": null
  }
},
```

**Example VS Code `settings.json`**
```javascript
...
    "quokka.darkTheme.log.decorationAttachmentRenderOptions": {
        "border": null,
        "borderColor": null,
        "fontStyle": null,
        "fontWeight": null,
        "textDecoration": null,
        "color": "#00ff00",
        "backgroundColor": null,
        "margin": "1.2em",
        "width": null,
        "height": null
    }
...
```
